tests/run-tests.py

```diff
@@ -1281,6 +1281,7 @@ def main():
                     "unicode",
                     "cmdline",
                     "ports/unix",
+                    "thread",
```

Comment:
I guess it makes sense to add this here.  The tests are actually run on the unix port as port of CI, but it's done separately, see line 268 of `ports/unix/Makefile`.  So probably that line should now be removed, otherwise the thread tests are run twice (on CI at least).

Reply:
It did break a few other tests in CI such as the Windows port which also uses this set, so I've reverted it.